css/src/metabox.css

```diff
@@ -1,6 +1,7 @@
 @import url(../src/icons.css);
 @import url(../src/modal.css);
 @import "../../node_modules/draft-js-mention-plugin/lib/plugin.css";
+@import "../../node_modules/@draft-js-plugins/mention/lib/plugin.css";
```

Comment:
I know, not invented in this PR. However, we should have this CSS in the replacement variable package instead.
Then we don't need this dependency here.

Reply:
The relative import cannot be used in the Javascript package because the position of `@draft-js-plugins/mention` changes when that package is added as a dependency. This is, again, due to the strange hoisting behavior, I think. 

I think we should fix the relative import (potentially the need to import css at all), the hoisting behavior, or both.